src/docs/asciidoc/reference/concepts/zetapush-npm-modules.adoc

```diff
@@ -1,2 +1,69 @@
 :relative-path: ../../
 include::{docdir}/variables.adoc[]
+
+ZetaPush is a *monorepo* that contains several *npm modules*. Each of them have a specific role.
```

Comment:
"monorepo": I think our clients don't care about that, it's our internal management.
We should simply say: "ZetaPush provides several npm modules for helping you in your development."

Each of them have a specific role. -> Each module has a specific role